rollup.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/bash
+# This script aggregates content
+set -xeuo pipefail
+mkdir -p /srv/tree
+cd /srv/tree
+ostree --repo=repo init --mode=archive
+ostree --repo=repo remote add remote --set=gpg-verify=false ${OSTREE_REPO_URL:-}
+ostree --repo=repo pull --mirror --depth=3 remote
```

Comment:
`--depth=3` is an interesting choice. The benefits make sense for a "devel" stream like this one since there's not a lot of change between commits. Or is this the idea for the "stable" ref as well (once we have one)?

Reply:
We're pulling all refs today...but anyways IMO we should switch to rojig and at that point there aren't refs, and we'll need to figure out about channels for rpms.